### PR TITLE
Community project files and CI starting with formatting

### PR DESCRIPTION
**Pull request by markstur**
By default, check black and isort on Python 3.8 (minimum supported version). Provide additional tasks for auto-fix and other python versions. Provide tasks for future lint (not passing yet) and tests.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Security Policy
+
+The Caikit project has a [common security policy that can be found here](https://github.com/caikit/community/blob/main/SECURITY.md).
```
Comment (hickeyma) [inline SECURITY.md:3]: Caikit project --> Caikit community

File: code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Community Code of Conduct
+
+The Caikit project [has a common code of conduct that can be found here](https://github.com/caikit/community/blob/main/code-of-conduct.md).
```
Comment (hickeyma) [inline code-of-conduct.md:3]: Caikit project --> Caikit community

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#####################################################
+#
+# List of approvers for caikit/caikit repository
```
Comment (hickeyma) [inline CODEOWNERS:3]: caikit/caikit --> caikit/caikit-huggingface-demo

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+When contributing, it's useful to start by looking at [issues](https://github.com/caikit/caikit-huggingface-demo/issues). After picking up an issue, writing code, or updating a document, make a pull request and your work will be reviewed and merged. If you're adding a new feature or find a bug, it's best to [write an issue](https://github.com/caikit/caikit-huggingface-demo/issues/new?assignees=&labels=&template=feature_request.md&title=) first to discuss it with maintainers.
+
+To contribute to this repo, you'll use the Fork and Pull model common in many open source repositories. For details on this process, check out [The GitHub Workflow
+Guide](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)
+from Kubernetes.
+
+When your contribution is ready, you can create a pull request. Pull requests are often referred to as "PR". In general, we follow the standard [GitHub pull request](https://help.github.com/en/articles/about-pull-requests) process. Follow the template to provide details about your pull request to the maintainers.
+
+Before sending pull requests, make sure your changes pass formatting, linting and unit tests.
```
Comment (hickeyma) [inline CONTRIBUTING.md:31]: Do unit tests exists?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+#### Code Review
+
+Once you've [created a pull request](#how-can-i-contribute), maintainers will review your code and may make suggestions to fix before merging. It will be easier for your pull request to receive reviews if you consider the criteria the reviewers follow while working. Remember to:
+
+- Run tests locally and ensure they pass
```
Comment (hickeyma) [inline CONTRIBUTING.md:37]: Do tests exists?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+## Your First Code Contribution
+
+Unsure where to begin contributing? You can start by looking through these issues:
+
+- Issues with the [`good first issue` label](https://github.com/caikit/caikit-huggingface-demo/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - these should only require a few lines of code and are good targets if you're just starting contributing.
+- Issues with the [`help wanted` label](https://github.com/caikit/caikit-huggingface-demo/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) - these range from simple to more complex, but are generally things we want but can't get to in a short time frame.
+
+## Releasing (Maintainers only)
+
+The responsibility for releasing new versions of the libraries falls to the maintainers. Releases will follow standard [semantic versioning](https://semver.org/).
```
Comment (hickeyma) [inline CONTRIBUTING.md:126]: Not applicable

**Review by hickeyma — CHANGES_REQUESTED** [review]

**Comment by markstur** [comment]
addressed review comments

**Review by hickeyma — APPROVED** [review]

**Pull request merged**